core/src/banking_stage/packet_receiver.rs

```diff
@@ -139,6 +149,27 @@ impl PacketReceiver {
             .swap(unprocessed_transaction_storage.len(), Ordering::Relaxed);
     }
 
+    fn filter_packets(
+        invalid_fee_payer_filter: &InvalidFeePayerFilter,
+        deserialized_packets: Vec<ImmutableDeserializedPacket>,
+    ) -> (Vec<ImmutableDeserializedPacket>, usize) {
+        let original_length = deserialized_packets.len();
+        let filtered_packets = deserialized_packets
+            .into_iter()
+            .filter(|packet| {
```

Comment:
remove any known invalid-fee-payer when receiving from sigverify into banking stage.

This prevents them from taking up buffer space